CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Documents changes that result in:
 
 ## Unreleased
 
+- Added contract_manager.get_contracts_deployed_info() that takes a module (`services`, `raiden` or `all`) instead of `services:bool`
```

Comment:
Suggestion: Add issue numbers to the changelog too? Would be nice to track.

Reply:
The PR number will be added here at the release time usually.

Reply:
I think it's easier to add the issue each time you make an addition. This is how we do it in Raiden at the moment, but it's up to you.

Reply:
For now I stick with old habits.

Reply:
Fair enough my friend :)